Content.Server/Shuttles/Commands/FTLDiskCommand.cs

```diff
@@ -0,0 +1,208 @@
+using Content.Server.Administration;
+using Content.Server.Construction.Completions;
+using Content.Server.Labels;
+using Content.Shared.Administration;
+using Content.Shared.Hands.Components;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Shuttles.Components;
+using Content.Shared.Storage;
+using Content.Shared.Storage.EntitySystems;
+using Robust.Shared.Console;
+using Robust.Shared.Map.Components;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Shuttles.Commands;
+
+/// <summary>
+/// Creates FTL disks, to maps, grids, or entities.
+/// </summary>
+[AdminCommand(AdminFlags.Fun)]
+
+public sealed class FTLDiskCommand : LocalizedCommands
+{
+    [Dependency] private readonly IEntityManager _entManager = default!;
+    [Dependency] private readonly IEntitySystemManager _entSystemManager = default!;
+
+    public override string Command => "ftldisk";
+
+    [ValidatePrototypeId<EntityPrototype>]
+    public const string CoordinatesDisk = "CoordinatesDisk";
+
+    [ValidatePrototypeId<EntityPrototype>]
+    public const string DiskCase = "DiskCase";
+    public override void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (args.Length == 0)
+        {
+            shell.WriteError("Not enough arguments.");
```

Comment:
Requires localization (see https://github.com/space-wizards/space-station-14/blob/2a22964e512eecb06d8b7c96e92c1b1dc2ffcc75/Resources/Locale/en-US/shell.ftl for some generic ones)